src/math/calculation.js

```diff
@@ -111,8 +111,10 @@ p5.prototype.constrain = function(n, low, high) {
  * @method dist
  * @param  {Number} x1 x-coordinate of the first point
  * @param  {Number} y1 y-coordinate of the first point
+ * @param  {Number} z1 z-coordinate of the first point (optional)
```

Comment:
rather than writing (optional), we put brackets around the optional argument (ex: [z1])